python/ray/rllib/__init__.py

```diff
@@ -16,6 +18,14 @@
 from ray.rllib.evaluation.sample_batch import SampleBatch
 
 
+def _setup_logger():
+    logger = logging.getLogger("ray.rllib")
+    handler = logging.StreamHandler()
+    handler.setFormatter(logging.Formatter("%(levelname)s: %(message)s"))
```

Comment:
I think there was some prior complaint about doing this, but it's probably fine...

Reply:
This is scoped to just rllib. However, we should consider adding timestamps and info everywhere, it's hard to debug stuff without the basics.

Reply:
Yeah, I understand; the complaint iirc was that the user should be in full control of the handlers that are provided to the logger (but that might be not be reasonable to expect for RLlib users), so it's fine